pkg/cmd/image/inspect.go

```diff
@@ -97,14 +97,15 @@ func Inspect(ctx context.Context, client *containerd.Client, identifiers []strin
 	defer cancel()
 
 	// Will hold the final answers
+	var errs []string
 	var entries []interface{}
 
 	snapshotter := containerdutil.SnapshotService(client, options.GOptions.Snapshotter)
 	// We have to query per provided identifier, as we need to post-process results for the case name + digest
 	for _, identifier := range identifiers {
 		candidateImageList, requestedName, requestedTag, err := inspectIdentifier(ctx, client, identifier)
 		if err != nil {
-			log.G(ctx).WithError(err).WithField("identifier", identifier).Error("failure calling inspect")
+			errs = append(errs, fmt.Sprintf("invalid reference format: %s", identifier))
```

Comment:
Is `invalid reference format` the only possible error here? Or we should return the (wrapped) original error? e.g., something like `fmt.Errorf("invalid reference %s: %w", identifier, err)`

Reply:
It looks like other errors from `inspectIdentifier` are errors returned from containerd image service. i.e. https://github.com/austinvazquez/nerdctl/blob/58fb7731a84ac3f260d971f8822aee8329bf1bbf/pkg/cmd/image/inspect.go#L62

So wrapping is useful in that case, but no in the case the error is invalid reference format. Perhaps it is best to revert back to log statement? I originally brought this in as error to align more with Docker behavior, but maybe it isn't worth losing the information here when error talking to containerd.
```
austin@nimbus:~/workspace/nerdctl$ mysudo nerdctl image inspect ∞∞∞∞∞∞∞∞∞∞ dne:latest
FATA[0000] 2 errors:
invalid reference format: ∞∞∞∞∞∞∞∞∞∞: invalid reference format
no such image: dne:latest 
austin@nimbus:~/workspace/nerdctl$ 
```

Reply:
How about let's do `fmt.Errorf("%w: %s", err, identifier)` 🤔?

The output will be something like:

```
invalid reference format: ∞∞∞∞∞∞∞∞∞∞
no such image: dne:latest 
containerd image service failed: containerd_err: dne:latest2
```

Reply:
Thanks for the suggestion btw; that solution was way simpler.